### ISSUE TITLE
Steam Bot Errors

### ISSUE THREAD

**Issue by thecsgobank** [ztizzlegaming/CSGOWinBig-SteamBot#22]
Hi i keep getting errors for my bot. If anyone could help thatd be awesome!
Here is the error log:

    SUCCESS: Schema Downloaded!
    [WinBig! 2015-10-11 17:37:25] SUCCESS: Steam Bot Logged In Completely!
    [WinBig! 2015-10-11 17:37:25] INFO: TradeOffer notification
    [WinBig! 2015-10-11 22:11:59] INFO: Connecting...
    [WinBig! 2015-10-11 22:11:59] SUCCESS: Done Loading Bot!
    [WinBig! 2015-10-11 22:12:04] SUCCESS: User Authenticated!
    [WinBig! 2015-10-11 22:12:04] ERROR: Offer declined because the offer wasn't a gift; the user wanted items instead of giving.
    [WinBig! 2015-10-11 22:12:06] ERROR: URI: https://steamcommunity.com/tradeoffer/765651288/decline >> System.Net.WebException: The remote server returned an error: (500) Internal Server Error.
       at System.Net.HttpWebRequest.GetResponse()
       at SteamTrade.SteamWeb.Request(String url, String method, NameValueCollection data, Boolean ajax, String referer) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamTrade\SteamWeb.cs:line 85
       at SteamTrade.SteamWeb.Fetch(String url, String method, NameValueCollection data, Boolean ajax, String referer) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamTrade\SteamWeb.cs:line 26
   at SteamTrade.TradeOffer.OfferSession.Decline(String tradeOfferId) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamTrade\TradeOffer\OfferSession.cs:line 83
       at SteamTrade.TradeOffer.TradeOffer.Decline() in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamTrade\TradeOffer\TradeOffer.cs:line 198
       at SteamBot.DepositTradeOfferUserHandler.OnNewTradeOffer(TradeOffer offer) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamBot\DepositTradeOfferUserHandler.cs:line 115
       at SteamBot.Bot.TradeOfferRouter(TradeOffer offer) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamBot\Bot.cs:line 846
       at SteamTrade.TradeOffer.TradeOfferManager.SendOfferToHandler(Offer offer) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamTrade\TradeOffer\TradeOfferManager.cs:line 142
       at SteamTrade.TradeOffer.TradeOfferManager.GetActiveTradeOffers() in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamTrade\TradeOffer\TradeOfferManager.cs:line 48
       at SteamTrade.TradeOffer.TradeOfferManager.GetOffers() in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamTrade\TradeOffer\TradeOfferManager.cs:line 121
   at SteamBot.Bot.UserWebLogOn() in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamBot\Bot.cs:line 736
       at SteamBot.Bot.<HandleSteamMessage>b__5(LoginKeyCallback callback) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamBot\Bot.cs:line 462
       at SteamKit2.CallbackMsgExtensions.Handle[T](ICallbackMsg msg, Action`1 handler)
       at SteamBot.Bot.HandleSteamMessage(ICallbackMsg msg) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamBot\Bot.cs:line 458
      at SteamBot.Bot.BackgroundWorkerOnDoWork(Object sender, DoWorkEventArgs doWorkEventArgs) in c:\Users\Denny\Desktop\CSGOWinBig-SteamBot\SteamBot\Bot.cs:line 941

**Comment by thecsgobank** [comment]
Also it says there was a trade notification when the bot has not gotten a trade notification...

**Comment by TheAnthonyNL** [comment]
Sometimes steam bugs out on 500 error ... im still trying to fix this somehow. On the other note the trade offer notification can be bugging sometimes just like in steam client when a trade offer is sent but items got traded away by other user.

**Comment by iLogix** [comment]
ERROR: Offer declined because the offer wasn't a gift; the user wanted items instead of giving.
this also seems to be a problem there. :)

**Comment by TheAnthonyNL** [comment]
It is not an error... its written in the bot. Of it wasnt there your bot would lose all skins for nothing.

**Comment by thecsgobank** [comment]
yes i am trying to figure out why it thinks it has received a trade offer when it clearly has not...
